events/python/receive_kombu.py

```diff
@@ -0,0 +1,35 @@
+from kombu import Consumer, Exchange, Queue, Message
+
+from setup_kombu import (
+    ARTICLE_EXCHANGE_NAME,
+    DASHBOARD_QUEUE_NAME,
+    get_connection
+)
+
+
+def message_handler(body: str, message: Message) -> None:
+    print(f'[x] Received: {body}')
+    message.ack()
+
+
+if __name__ == '__main__':
+    with get_connection() as connection:
+
+        with connection.channel() as channel:
+
+            queue = Queue(name=DASHBOARD_QUEUE_NAME,
+                          exchange=Exchange(name=ARTICLE_EXCHANGE_NAME, type='fanout'))
```

Comment:
I wouldn't expect to have to know anything about the `Exchange` from a consumer point of view, only the queue name :thinking:

Reply:
That was what I expected. Though if you remove the exchange arg, you receive no messages. Would require a further dip into the docs.

Reply:
I have seen in several places in the docs it wants consumers to define the `exchange` as well as the `queue` you want to consume from. Possibly a side effect of them supporting multiple transports? 🤔 whatever the reason I don't like it. Another way to use some of there mixin classes but that looks overly verbose for what is needed.